docs/output.md

```diff
@@ -1,72 +1,97 @@
-# nf-core/sarek: Output
+# nf-core/sarek: Output <!-- omit in toc -->
```

Comment:
Uuh! Nice :-)

Reply:
I did just read the docs of the tool you advised to try ;-)